runtime/lua/vim/lsp/client.lua

```diff
@@ -387,6 +387,7 @@ function Client.create(config)
     capabilities = config.capabilities,
     workspace_folders = lsp._get_workspace_folders(config.workspace_folders or config.root_dir),
     root_dir = config.root_dir,
+    is_stopping = false,
```

Comment:
`is_stopped()` is already a public method, don't need another public field. Ideally we wouldn't need this field at all, but I'm assuming there is no other way.
```suggestion
    _is_stopping = false,
```

Reply:
yes, makes sense thx.